shared/css/typography.scss

```diff
@@ -37,37 +37,37 @@
 @mixin heading-xxl {
   @include heading-common;
   font-family: $barlowSemiCondensed-semibold;
-  font-size: 3em;
```

Comment:
Yeah definitely! I can add it at the top of the doc w/ the other comments. 

And re: the margins being in `em` this is actually a good example of when we want the margin to be relative to the parent element rather than the root. If you change it to `rem` it'll set the bottom margin to `8px` on all of the headings, instead of varying it based on font-size. 

`rem` works great for structural element spacing like `section` and `div` though.

Reply:
Any chance you could do a small presentation to our team about when we should use `rem` and `em`? Sounds like I should start using `rem` most of the time, unless I want relative sizing

Reply:
Absolutely!